src/main/java/com/acrolinx/sidebar/utils/MatchUtils.java

```diff
@@ -51,6 +52,23 @@ public static List<? extends AbstractMatch> filterDangerousMatches(
         .collect(Collectors.toList());
   }
 
+  public static List<? extends AbstractMatch> filterInsignificantWhitespaceMatches(
+      List<? extends AbstractMatch> abstractMatches, String lastCheckedContent) {
+    return abstractMatches.stream()
+        .filter(
+            match -> {
```

Comment:
since the type is called "AbstractMatch", I suggest to call the variable here "abstractMatch" too.

Reply:
Fixed